manifests/cf-manifest/manifest/020-cf-properties.yml

```diff
@@ -613,3 +613,8 @@ properties:
 
   garden:
     log_level: error
+  racoon:
+    certificate_authority_cert: (( grab secrets.bosh_ca_cert ))
+    certificate_authority_private_key: (( grab secrets.bosh_ca_key ))
```

Comment:
Actually the CA private key is deployed [when the job is updated](https://github.com/SAP/ipsec-release/blob/1d27c9115adc838d0cfac402a6615d0dbf16a912/jobs/racoon/templates/etc/racoon/certs/ca_pkey.pem.erb) but [deleted when racoon starts](https://github.com/SAP/ipsec-release/blob/master/jobs/racoon/templates/bin/racoon_ctl.erb#L36-L54) (and the local certificate is recreated).
So it's not that bad.